frontend/device/pocketbook/device.lua

```diff
@@ -506,6 +506,16 @@ local PocketBook617 = PocketBook:extend{
     hasNaturalLight = yes,
 }
 
+-- PocketBook Basic Lux 4 (618)
+local PocketBook618 = PocketBook:extend{
+    model = "PB618",
+    display_dpi = 212,
+    isTouchDevice = yes,
+    hasDPad = no,
+    hasFewKeys = yes,
+    hasFrontlight = yes,
```

Comment:
Just checking but the device does not have support for natural light like the previous touch lux?

Reply:
> Just checking but the device does not have support for natural light like the previous touch lux?

Only white LEDs are installed in the PB618, PB617 has white and orange LEDs.

![IMG_20231008_101741_476](https://github.com/koreader/koreader/assets/84080014/6b37f002-f189-42c6-b425-5252a05e0068)

PB618 has a touchscreen, but PB617 does not. That's why PB617 has a D-pad, but PB618 doesn't.